crypto/rand/rand_win.c

```diff
@@ -567,51 +166,6 @@ int RAND_poll(void)
     return (1);
 }
 
-int RAND_event(UINT iMsg, WPARAM wParam, LPARAM lParam)
```

Comment:
Yes, I think we keep them but deprecated (using the OPENSSL_API_COMPAT macro as @richsalz suggests), and all they do is call RAND_poll/RAND_status.